packages/utils/src/signatures.ts

```diff
@@ -0,0 +1,92 @@
+import { AuthTypes } from "@walletconnect/types";
+import { hashMessage } from "@ethersproject/hash";
+import { recoverAddress } from "@ethersproject/transactions";
+export const DEFAULT_RPC_URL = "https://rpc.walletconnect.com/v1";
```

Comment:
Belongs in `constants.ts` if being exported/is non-local to this util

Reply:
removed the `export` so it stays local